bindings/python/libmata.pyx

```diff
@@ -1116,7 +1177,8 @@ cdef class Nfa:
 
     @classmethod
     def equivalence_check(cls, Nfa lhs, Nfa rhs, Alphabet alphabet = None, params = None) -> bool:
-        """Test equivalence of two automata.
+        """
+        Test equivalence of two automata.
```

Comment:
Did we not agree on having the brief description already on the first line with `"""`?

Reply:
Yes, probably type. Thanks, will fix this :-)